app/views/organizations/show.html.erb

```diff
@@ -1,49 +1,117 @@
 <%= render "individual_actions" %>
 
-<p>
-  <strong>Name:</strong>
-  <%= @organization.name %>
-</p>
+<div class="container">
 
-<p>
-  <strong>Description:</strong>
-  <%= @organization.description %>
-</p>
+<h2><%= @organization.name %></h2>
+<div class="organization-url"><%= @organization.url %></div>
 
-<p>
-  <strong>Address:</strong>
-  <span class="org-address"><%= @organization.address %></span>
-</p>
+<!-- column 1 -->
+  <div id="col1">
 
-<p>
-  <strong>Phone:</strong>
-  <% @organization.phones.each do |phone| %>
-    <%= phone %>
-  <% end %>
-</p>
+  <!-- Services -->
+    <div class="border-above">
+      <h4>Services</h4>
+        <ul>
+          <!-- Primary Services -->
+          <li>
+            <ul id="primary-services">
+            <li>Substance Abuse Treatment </li>
+            <li>Mental Health Treatment </li>
+            <li>Health & Wellness Education </li>
+          </ul>
+        </li>
+        <!-- Additional Services -->
+        <li>
+          <ul class="chunked">
+            <li id="communication">Internet, Phone, Voicemail</li>
+               <li id="food">Meals</li>
+               <li id="goods">Personal Care Items</li>
+               <li id="legal">Referrals</li>
+               <li id="health">Anger Management, Dual Diagnosis Treatment, Group Counseling, Individual Counseling</li>
+               <li id="transit">Vouchers</li>
+               <li id="employment">Job Readiness/Life Skills, Post-Incarceration Support</li>
+           </ul>
+        </li>
+      </ul>
+      </div>
 
-<p>
-  <strong>Email:</strong>
-  <% @organization.emails.each do |email| %>
-    <%= email %>
-  <% end %>
-</p>
+  <!-- Eligbility and Appropriateness -->
+    <div class="border-above">
+      <h4>Is this for me?</h4>
+        <!-- Eligbility -->
+        <h5>Serves</h5>
+        <div>Men, Women, ages 12-25, involved in criminal justice system.</div>
+        <!-- Fees -->
+        <h5>Fees</h5>
+        <div>None.</div>
+        <!-- Languages -->
+        <h5>Languages</h5>
+        <div><%= @organization.languages.join(", ") %></div>
+        <!-- Accessibility -->
+        <h5>Accessibility</h5>
+        <div>Wheelchair</div>
+        <!-- Faith -->
+        <h5>Faith Based</h5>
+        <div>No</div>
+      <!-- About -->
+        <h5>About</h5>
+        <div>
+        <%= @organization.description %>
+        </div>
+    </div>
+</div>
 
-<p>
-  <strong>Url:</strong>
-  <%= @organization.url %>
+<!-- column 2 -->
+  <div id="col2">
+  <!-- Location -->
+  <div class="border-around">
+    <div class="map" id="map-canvas" style="height: 300px;"></div>
```

Comment:
My fault for adding this quickly originally, but height should be set in the stylesheet.